dev-tools/aws_creds_from_vault.sh

```diff
@@ -42,12 +42,17 @@ case $- in
         ;;
 esac
 
-export VAULT_TOKEN=$(vault write -field=token auth/approle/login role_id="$VAULT_ROLE_ID" secret_id="$VAULT_SECRET_ID")
+PREFIX=""
+if [ "$BUILDKITE" = true ]; then
+    PREFIX="secret/ci/elastic-ml-cpp/"
+else
+    export VAULT_TOKEN=$(vault write -field=token auth/approle/login role_id="$VAULT_ROLE_ID" secret_id="$VAULT_SECRET_ID")
```

Comment:
Does this imply BuildKite automatically sets `VAULT_TOKEN`?

Reply:
Yes, behind the scenes access to CI vault is automagically set up for us.